receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -147,8 +147,8 @@ func (ghs *githubScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 			}
 
 			// When enabled, process any CVEs for the repository
-			if ghs.cfg.Metrics.GitRepositoryCveCount.Enabled {
-				cves, err := ghs.getCVEs(ctx, genClient, name)
+			if ghs.cfg.Metrics.GitRepositoryCveCount.Enabled && name == "liatrio-otel-collector" {
```

Comment:
```suggestion
			if ghs.cfg.Metrics.GitRepositoryCveCount.Enabled {
```

There shouldn't be a reference in a conditional to `liatrio-otel-collector`. I presume this was just a local test thing.

Reply:
Correct, will be removed when pr is taken out of draft

Reply:
Removed and pushed changes up

Reply:
Marking as resolved